src/azure-cli/service_name.json

```diff
@@ -419,6 +424,11 @@
     "AzureServiceName": "Azure Resource Manager",
     "URL": "https://docs.microsoft.com/azure/azure-resource-manager/management/overview"
   },
+  {
+    "Command": "az resourcemanagement",
+    "AzureServiceName": "Azure Resource Manager",
+    "URL": "https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/create-private-link-access-portal"
+  },
```

Comment:
> https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/create-private-link-access-portal

Are you sure the document link of `az resourcemanagement` is `create-private-link-access-portal`?

Reply:
Got confirmation from @jennyhunter-msft that 
>https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/create-private-link-access-portal

Is the correct document link to use for now

Reply:
Out of curiosity, shouldn't the private-link be just a part of resource management? Shouldn't [Azure Resource Manager documentation | Microsoft Docs](https://docs.microsoft.com/en-us/azure/azure-resource-manager/) be a more appropriate doc link for the `az resourcemanagement` command?

Reply:
The Private Link work is a subsection of resource management with several specific constraints. As far as helpfulness to the user, the private link specific documentation would be more useful.

Reply:
OK, thanks~